src/pages/Help.vue

```diff
@@ -1,3 +1,36 @@
+<template>
```

Comment:
Please never remove functionality. You extracted a template into BlockCard, please also use it now.